[2021-12-28 13:43:20,773] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_sligo.persist_data scheduled__2021-12-27T10:45:00+00:00 [queued]>
[2021-12-28 13:43:20,874] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_sligo.persist_data scheduled__2021-12-27T10:45:00+00:00 [queued]>
[2021-12-28 13:43:20,880] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:43:20,901] {taskinstance.py:1242} INFO - Starting attempt 2 of 4
[2021-12-28 13:43:20,902] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:43:20,985] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): persist_data> on 2021-12-27 10:45:00+00:00
[2021-12-28 13:43:21,076] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_sligo', 'persist_data', 'scheduled__2021-12-27T10:45:00+00:00', '--job-id', '5105', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_sligo.py', '--cfg-path', '/tmp/tmphifja6m5', '--error-file', '/tmp/tmp62ncygka']
[2021-12-28 13:43:21,125] {standard_task_runner.py:77} INFO - Job 5105: Subtask persist_data
[2021-12-28 13:43:21,004] {standard_task_runner.py:52} INFO - Started process 54169 to run task
[2021-12-28 13:43:21,742] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_sligo.persist_data scheduled__2021-12-27T10:45:00+00:00 [running]> on host 0c125a5bba95
[2021-12-28 13:43:22,407] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_sligo
AIRFLOW_CTX_TASK_ID=persist_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-27T10:45:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-27T10:45:00+00:00
[2021-12-28 13:43:22,504] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-28 13:43:22,514] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name persist data --verbose /opt/***/jobs/persist_db.py sligo
[2021-12-28 13:43:30,167] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-28 13:43:30,509] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-28 13:43:30,510] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-28 13:43:30,511] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-28 13:43:30,513] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-28 13:43:30,513] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-28 13:43:30,742] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-28 13:43:30,744] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-28 13:43:30,744] {spark_submit.py:514} INFO - deployMode              null
[2021-12-28 13:43:30,746] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-28 13:43:30,747] {spark_submit.py:514} INFO - executorCores           null
[2021-12-28 13:43:30,748] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-28 13:43:30,749] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-28 13:43:30,750] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-28 13:43:30,751] {spark_submit.py:514} INFO - driverCores             null
[2021-12-28 13:43:30,752] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-28 13:43:30,753] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-28 13:43:30,755] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-28 13:43:30,756] {spark_submit.py:514} INFO - supervise               false
[2021-12-28 13:43:30,757] {spark_submit.py:514} INFO - queue                   null
[2021-12-28 13:43:30,758] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-28 13:43:30,759] {spark_submit.py:514} INFO - files                   null
[2021-12-28 13:43:30,760] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-28 13:43:30,761] {spark_submit.py:514} INFO - archives                null
[2021-12-28 13:43:30,766] {spark_submit.py:514} INFO - mainClass               null
[2021-12-28 13:43:30,768] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/persist_db.py
[2021-12-28 13:43:30,771] {spark_submit.py:514} INFO - name                    persist data
[2021-12-28 13:43:30,772] {spark_submit.py:514} INFO - childArgs               [sligo]
[2021-12-28 13:43:30,773] {spark_submit.py:514} INFO - jars                    null
[2021-12-28 13:43:30,775] {spark_submit.py:514} INFO - packages                null
[2021-12-28 13:43:30,776] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-28 13:43:30,777] {spark_submit.py:514} INFO - repositories            null
[2021-12-28 13:43:30,778] {spark_submit.py:514} INFO - verbose                 true
[2021-12-28 13:43:30,780] {spark_submit.py:514} INFO - 
[2021-12-28 13:43:30,781] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-28 13:43:30,782] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-28 13:43:30,783] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:43:30,785] {spark_submit.py:514} INFO - 
[2021-12-28 13:43:30,786] {spark_submit.py:514} INFO - 
[2021-12-28 13:43:32,113] {spark_submit.py:514} INFO - Main class:
[2021-12-28 13:43:32,115] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-28 13:43:32,119] {spark_submit.py:514} INFO - Arguments:
[2021-12-28 13:43:32,128] {spark_submit.py:514} INFO - file:/opt/***/jobs/persist_db.py
[2021-12-28 13:43:32,129] {spark_submit.py:514} INFO - null
[2021-12-28 13:43:32,130] {spark_submit.py:514} INFO - sligo
[2021-12-28 13:43:32,131] {spark_submit.py:514} INFO - --verbose
[2021-12-28 13:43:32,139] {spark_submit.py:514} INFO - Spark config:
[2021-12-28 13:43:32,140] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:43:32,141] {spark_submit.py:514} INFO - (spark.app.name,persist data)
[2021-12-28 13:43:32,142] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-28 13:43:32,143] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-28 13:43:32,144] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-28 13:43:32,145] {spark_submit.py:514} INFO - 
[2021-12-28 13:43:32,146] {spark_submit.py:514} INFO - 
[2021-12-28 13:43:32,147] {spark_submit.py:514} INFO - 
[2021-12-28 13:43:35,363] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-28 13:43:35,386] {spark_submit.py:514} INFO - 21/12/28 13:43:35 INFO SparkContext: Running Spark version 3.2.0
[2021-12-28 13:43:35,568] {spark_submit.py:514} INFO - 21/12/28 13:43:35 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-28 13:43:35,963] {spark_submit.py:514} INFO - 21/12/28 13:43:35 INFO ResourceUtils: ==============================================================
[2021-12-28 13:43:35,964] {spark_submit.py:514} INFO - 21/12/28 13:43:35 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-28 13:43:35,965] {spark_submit.py:514} INFO - 21/12/28 13:43:35 INFO ResourceUtils: ==============================================================
[2021-12-28 13:43:35,966] {spark_submit.py:514} INFO - 21/12/28 13:43:35 INFO SparkContext: Submitted application: Persist DB
[2021-12-28 13:43:36,033] {spark_submit.py:514} INFO - 21/12/28 13:43:36 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-28 13:43:36,060] {spark_submit.py:514} INFO - 21/12/28 13:43:36 INFO ResourceProfile: Limiting resource is cpu
[2021-12-28 13:43:36,061] {spark_submit.py:514} INFO - 21/12/28 13:43:36 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-28 13:43:36,241] {spark_submit.py:514} INFO - 21/12/28 13:43:36 INFO SecurityManager: Changing view acls to: ***
[2021-12-28 13:43:36,242] {spark_submit.py:514} INFO - 21/12/28 13:43:36 INFO SecurityManager: Changing modify acls to: ***
[2021-12-28 13:43:36,244] {spark_submit.py:514} INFO - 21/12/28 13:43:36 INFO SecurityManager: Changing view acls groups to:
[2021-12-28 13:43:36,245] {spark_submit.py:514} INFO - 21/12/28 13:43:36 INFO SecurityManager: Changing modify acls groups to:
[2021-12-28 13:43:36,246] {spark_submit.py:514} INFO - 21/12/28 13:43:36 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-28 13:43:37,575] {spark_submit.py:514} INFO - 21/12/28 13:43:37 INFO Utils: Successfully started service 'sparkDriver' on port 44455.
[2021-12-28 13:43:37,838] {spark_submit.py:514} INFO - 21/12/28 13:43:37 INFO SparkEnv: Registering MapOutputTracker
[2021-12-28 13:43:38,005] {spark_submit.py:514} INFO - 21/12/28 13:43:38 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-28 13:43:38,078] {spark_submit.py:514} INFO - 21/12/28 13:43:38 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-28 13:43:38,085] {spark_submit.py:514} INFO - 21/12/28 13:43:38 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-28 13:43:38,121] {spark_submit.py:514} INFO - 21/12/28 13:43:38 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-28 13:43:38,226] {spark_submit.py:514} INFO - 21/12/28 13:43:38 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-20d6f284-68f4-41dd-9491-34ea685af1e8
[2021-12-28 13:43:38,283] {spark_submit.py:514} INFO - 21/12/28 13:43:38 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-28 13:43:38,364] {spark_submit.py:514} INFO - 21/12/28 13:43:38 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-28 13:43:38,911] {spark_submit.py:514} INFO - 21/12/28 13:43:38 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2021-12-28 13:43:39,075] {spark_submit.py:514} INFO - 21/12/28 13:43:39 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://0c125a5bba95:4040
[2021-12-28 13:43:40,281] {spark_submit.py:514} INFO - 21/12/28 13:43:40 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:43:40,620] {spark_submit.py:514} INFO - 21/12/28 13:43:40 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:43:40,622] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:43:40,623] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:43:40,625] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:43:40,626] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:43:40,628] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:43:40,629] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:43:40,630] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:43:40,632] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:43:40,633] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:43:40,635] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:43:40,636] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:43:40,638] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:43:40,640] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:43:40,641] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:43:40,643] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:43:40,644] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:43:40,646] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:43:40,647] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:43:40,649] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:43:40,651] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:43:40,652] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:43:40,654] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:43:40,655] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:43:40,657] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:43:40,659] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:43:40,660] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:43:40,662] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:43:40,663] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:43:40,665] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:43:40,667] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:43:40,669] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:43:40,671] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:43:40,673] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:43:40,674] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:43:40,676] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:43:40,678] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:43:40,680] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:43:40,681] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:43:40,683] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:43:40,684] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:43:40,685] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:43:40,686] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:43:40,688] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:43:40,689] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:43:40,691] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:43:40,693] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:43:40,695] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:43:40,696] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:43:40,697] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:43:40,698] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:43:40,699] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:43:40,701] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:43:40,702] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:43:40,703] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:43:40,704] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:43:40,706] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:43:40,707] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:44:00,277] {spark_submit.py:514} INFO - 21/12/28 13:44:00 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:44:00,295] {spark_submit.py:514} INFO - 21/12/28 13:44:00 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:44:00,296] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:44:00,297] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:44:00,298] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:44:00,299] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:44:00,300] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:44:00,301] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:44:00,301] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:44:00,302] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:44:00,302] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:44:00,303] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:44:00,304] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:44:00,305] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:44:00,305] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:44:00,306] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:44:00,307] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:44:00,307] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:44:00,308] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:44:00,309] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:44:00,311] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:44:00,312] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:44:00,314] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:44:00,315] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:44:00,317] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:44:00,318] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:44:00,319] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:44:00,321] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:44:00,322] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:44:00,324] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:44:00,324] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:44:00,325] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:44:00,326] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:44:00,327] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:44:00,328] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:44:00,329] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:44:00,330] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:44:00,331] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:44:00,331] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:44:00,332] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:44:00,333] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:44:00,333] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:44:00,334] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:44:00,336] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:44:00,336] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:44:00,337] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:44:00,337] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:44:00,338] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:44:00,338] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:44:00,339] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:44:00,340] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:44:00,340] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:44:00,341] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:44:00,341] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:44:00,342] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:44:00,343] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:44:00,343] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:44:00,344] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:44:00,345] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:44:20,244] {spark_submit.py:514} INFO - 21/12/28 13:44:20 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:44:20,265] {spark_submit.py:514} INFO - 21/12/28 13:44:20 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:44:20,266] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:44:20,267] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:44:20,268] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:44:20,269] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:44:20,270] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:44:20,271] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:44:20,272] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:44:20,273] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:44:20,274] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:44:20,274] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:44:20,276] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:44:20,277] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:44:20,278] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:44:20,279] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:44:20,280] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:44:20,281] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:44:20,282] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:44:20,283] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:44:20,284] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:44:20,285] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:44:20,286] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:44:20,287] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:44:20,288] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:44:20,289] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:44:20,289] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:44:20,290] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:44:20,291] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:44:20,292] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:44:20,293] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:44:20,294] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:44:20,295] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:44:20,296] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:44:20,297] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:44:20,299] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:44:20,301] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:44:20,305] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:44:20,305] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:44:20,307] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:44:20,307] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:44:20,308] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:44:20,309] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:44:20,310] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:44:20,311] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:44:20,312] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:44:20,314] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:44:20,315] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:44:20,316] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:44:20,317] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:44:20,317] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:44:20,318] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:44:20,319] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:44:20,320] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:44:20,321] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:44:20,321] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:44:20,322] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:44:20,323] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:44:20,324] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:44:40,210] {spark_submit.py:514} INFO - 21/12/28 13:44:40 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-28 13:44:40,213] {spark_submit.py:514} INFO - 21/12/28 13:44:40 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-28 13:44:40,234] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 40531.
[2021-12-28 13:44:40,254] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO NettyBlockTransferService: Server created on 0c125a5bba95:40531
[2021-12-28 13:44:40,255] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-28 13:44:40,256] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO SparkUI: Stopped Spark web UI at http://0c125a5bba95:4040
[2021-12-28 13:44:40,257] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 0c125a5bba95, 40531, None)
[2021-12-28 13:44:40,258] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO BlockManagerMasterEndpoint: Registering block manager 0c125a5bba95:40531 with 434.4 MiB RAM, BlockManagerId(driver, 0c125a5bba95, 40531, None)
[2021-12-28 13:44:40,259] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-28 13:44:40,260] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 0c125a5bba95, 40531, None)
[2021-12-28 13:44:40,261] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-28 13:44:40,262] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 0c125a5bba95, 40531, None)
[2021-12-28 13:44:40,266] {spark_submit.py:514} INFO - 21/12/28 13:44:40 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-28 13:44:40,284] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-28 13:44:40,306] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO MemoryStore: MemoryStore cleared
[2021-12-28 13:44:40,307] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO BlockManager: BlockManager stopped
[2021-12-28 13:44:40,319] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-28 13:44:40,325] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-28 13:44:40,346] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO SparkContext: Successfully stopped SparkContext
[2021-12-28 13:44:40,492] {spark_submit.py:514} INFO - 21/12/28 13:44:40 ERROR SparkContext: Error initializing SparkContext.
[2021-12-28 13:44:40,492] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:44:40,493] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:44:40,494] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:44:40,495] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:44:40,496] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:44:40,496] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:44:40,497] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:44:40,498] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:44:40,499] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:44:40,499] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:44:40,500] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:44:40,503] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:44:40,504] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:44:40,506] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:44:40,507] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:44:40,507] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:44:40,508] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:44:40,508] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO SparkContext: SparkContext already stopped.
[2021-12-28 13:44:40,509] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-28 13:44:40,510] {spark_submit.py:514} INFO - File "/opt/***/jobs/persist_db.py", line 52, in <module>
[2021-12-28 13:44:40,510] {spark_submit.py:514} INFO - .config("spark.driver.bindAddress", "localhost") \
[2021-12-28 13:44:40,511] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-28 13:44:40,512] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-28 13:44:40,513] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-28 13:44:40,514] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-28 13:44:40,515] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-28 13:44:40,516] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-28 13:44:40,517] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-28 13:44:40,517] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-28 13:44:40,518] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:44:40,519] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:44:40,520] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:44:40,521] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:44:40,522] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:44:40,523] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:44:40,523] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:44:40,524] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:44:40,525] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:44:40,526] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:44:40,526] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:44:40,527] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:44:40,528] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:44:40,529] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:44:40,529] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:44:40,530] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:44:40,531] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:44:40,532] {spark_submit.py:514} INFO - 
[2021-12-28 13:44:40,600] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO ShutdownHookManager: Shutdown hook called
[2021-12-28 13:44:40,601] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO ShutdownHookManager: Deleting directory /tmp/spark-969be938-fd3f-4449-8875-8226208e1190
[2021-12-28 13:44:40,606] {spark_submit.py:514} INFO - 21/12/28 13:44:40 INFO ShutdownHookManager: Deleting directory /tmp/spark-79228beb-3b1c-46b9-8b66-8c713b80105d
[2021-12-28 13:44:40,723] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name persist data --verbose /opt/***/jobs/persist_db.py sligo. Error code is: 1.
[2021-12-28 13:44:40,731] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_sligo, task_id=persist_data, execution_date=20211227T104500, start_date=20211228T134320, end_date=20211228T134440
[2021-12-28 13:44:40,749] {standard_task_runner.py:91} ERROR - Failed to execute job 5105 for task persist_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name persist data --verbose /opt/***/jobs/persist_db.py sligo. Error code is: 1.
[2021-12-28 13:44:40,781] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-28 13:44:41,122] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
